tests/accounts/avalanche.test.ts

```diff
@@ -1,26 +1,21 @@
 import { avalanche, post } from "../index";
 import { DEFAULT_API_V2 } from "../../src/global";
-import { ItemType } from "../../src/messages/message";
+import { Chain, ItemType } from "../../src/messages/message";
 
 describe("Avalanche accounts", () => {
-    it("should create a new Avalanche account", async () => {
-        const { account } = await avalanche.NewAccount();
-
-        expect(account.address).not.toBe("");
-    });
-
-    it("should retreive an avalanche keypair from an hexadecimal private key", async () => {
+    it("should retrieve an avalanche keypair from an hexadecimal private key", async () => {
         const { account, privateKey } = await avalanche.NewAccount();
 
         if (privateKey) {
             const accountFromPK = await avalanche.ImportAccountFromPrivateKey(privateKey);
             expect(account.address).toBe(accountFromPK.address);
+            expect(account.GetChain()).toBe(Chain.AVAX);
```

Comment:
As getChain() is used for sending messages to Aleph, In my opinion, is a good thing to have this debug to be sure errors won't come from this.

I see more this test to help when you are adding functionalities to the account to be sure you don't make a typo somewhere and save some debugging time.

I don't know if you get my vision, but I understand it might look a bit overkill